startup.sh

```diff
@@ -14,4 +14,4 @@ then
     echo Adding ip route add "${LAN_NETWORK}" via "${DEFAULT_GATEWAY}" dev eth0 for attached container web ui access
     echo Do not forget to expose the ports for attached container we ui access
 fi
-openvpn --config $VPN_FILE --auth-user-pass vpn-auth.txt
+openvpn --config $VPN_FILE --auth-user-pass vpn-auth.txt --mute-replay-warnings $OPENVPN_OPTS
```

Comment:
Hi,
Really good the addition of $OPENVPN_OPTS env, BTW the mute replay warnings should be optional.
In my case, I would like to see them

Reply:
Maybe it wasn't completely clear what that option is for- without it, especially when running bittorrent (over UDP), the log is nothing *but* that message. It's millions of lines of the same warning.

I definitely see the motive for having things be optional but this flag really isn't optional, especially for the advertised ease of use with docker-compose & bittorrent.

In the circumstance that you'd still like it optional, would an optional environment variable to *enable* the warnings suffice?

Reply:
What if we define it as default value for OPENVPN_OPTS?

Reply:
I thought about suggesting that as well. I opted to just add it to the OpenVPN command line arguments because if someone wanted to add an option, they would then also have to be told (and to properly remember) to add the `--mute-replay-warnings` flag as well.

It just seemed like a lesser solution to the issue than just having it always enabled. After all, the warning itself is harmless.